src/components/CreateRules/UploadRules.vue

```diff
@@ -307,47 +383,62 @@ li {
   }
 }
 
-#confirm-rules-section {
+::v-deep #confirm-rules-section {
   #confirm-rules-card {
     margin-top: 15px;
-    padding-top: 5px;
-    padding-bottom: 20px;
+    padding-top: 40px;
+    padding-bottom: 40px;
     padding-left: 25px;
+    padding-right: 25px;
+  }
+
+  #approved-name {
+    font-weight: bold;
+    margin-top: -3px;
+    margin-left: 0px;
+  }
+
+  // override default validation styling so checkbox does not turn red on validation error
+  .v-input--selection-controls__input i {
+    color:rgba(0, 0, 0, 0.54) !important;
```

Comment:
Please define a new colour variable (in theme) and then use it here.

Reply:
so i was looking at this gray and i see that the gray palette is defined from lighter to darker.  if i follow that, it will push the current `gray7` to `gray8` and of course everything after that shifts too.  is this ok?

Reply:
I think those grays are official BC Gov theme colours, which obviously we cannot change. You'll probably have to come up with a new name reflecting its purpose. Add it to the "App Colors" section at the bottom of theme.scss.

Reply:
ok, sounds good. thanks

Reply:
Sometimes, it's not about implementing the "best" solution, but the "least worst" one! 😁

Reply:
added `$app-lt-gray` to `themes.css` and updated to reference in `UploadRules.vue` component

Reply:
Looks good. Great colour names!